### ISSUE TITLE
Full Volume Restore does not restore entire volume

### ISSUE THREAD

**Issue by mimitche** [NetAppDocs/occm#403]
Page: [Managing backups for Cloud Volumes ONTAP and on-premises ONTAP systems](https://docs.netapp.com/us-en/occm/task_managing_backups.html)

This page states: "Cloud Manager performs a full volume restore to a new volume."

The assumption based on this statement is the **entire** volume is restored as it looked when backed up, **including snapshots**.  The restore I just tried did not restore any of the snapshots; it only restored a single snapshot based on the backup I selected in Cloud Manager to restore.  The original volume was 48GB in size, but the restored volume is only 18GB.

If Cloud Backup is only going to restore a single point in time version of a volume that needs to be clarified.  The way it is currently worded the assumption is the full volume (including snapshots) is restored.

**Comment by netapp-tonacki** [comment]
Hi mimitche,  That's a good point to clarify.  I am checking with the project team and I'll make changes based on their feedback.

**Comment by netapp-tonacki** [comment]
Hi again mimitche,  It is correct that backups do not contain snapshots. The original volume will contain all of its' snapshots if needed. And the new volume (based on the backup) can start creating snapshots relevant to that new volume.  So I updated the topic [How Backup to Cloud works](https://docs.netapp.com/us-en/occm/concept_backup_to_cloud.html#how-backup-to-cloud-works) to mention that snapshot are not included in the backup, and I also updated the topic you identified to say no snapshots are "restored" to the new volume created from the backup.

Thanks for your comment to clarify it for others.
